tutorials/incorporating_oli_calculations.ipynb

```diff
@@ -284,7 +388,7 @@
    "source": [
     "# 7. Extract Filtered Output\n",
```

Comment:
Previous step was step 5, and now we skip to step 7. Change to step 6, or add step 6 in before this.

Reply:
Step 5 and 6 are in the same box, but I will make this more clear